python/src/uagents/config.py

```diff
@@ -115,3 +117,24 @@ def get_logger(logger_name, level=logging.INFO):
     logger.addHandler(log_handler)
     logger.propagate = False
     return logger
+
+
+def log(logger: Optional[logging.Logger], level: str, message: str):
+    if logger:
+        logger.log(level, message)
+
+
+def add_task(
+    loop: asyncio.AbstractEventLoop, task_list: Set[asyncio.Task], func: Callable
+):
+    """
+    Create a task and add it to the agents event loop and task list.
+
+    Args:
+        loop (asyncio.AbstractEventLoop): Agent event loop.
+        task_list (Set[asyncio.Task]): Agent internal task list.
+        func (Callable): Function to be executed as a task.
+    """
+    task = loop.create_task(func)
+    task_list.add(task)
+    task.add_done_callback(task_list.discard)
```

Comment:
These would seem to be a better fit for a new `utils.py` or `helpers.py` module.

Reply:
Good idea, moved the two functions over to `utils.py`